src/AzureAIProxy/Middleware/LoadProperties.cs

```diff
@@ -0,0 +1,68 @@
+using AzureAIProxy.Shared.Database;
+using System.Text.Json;
+
+namespace AzureAIProxy.Middleware;
+
+public class LoadProperties(RequestDelegate next)
+{
+    private readonly RequestDelegate _next = next;
```

Comment:
You don't need to capture these as a field of the type when using a primary constructor, you can access the `next` argument within `InvokeAsync`